justfile

```
@@ -19,6 +19,9 @@ cargo := "cargo" + if toolchain != "" { " +" + toolchain } else { "" }
 # The version name to use for packages.
 package_version := `git rev-parse --short HEAD`
 
+# Docker tag
+docker_tag := env_var_or_default("USER", "dev") + "/l2-proxy:" + package_version
```

another thought...I wonder if we want to include the branch name for this as well, since it's not intended to be used for actual releases?
```suggestion
docker_tag := env_var_or_default("USER", "dev") + "/l2-proxy:" + `git rev-parse --abbrev-ref HEAD` + "." package_version
```
or similar?

---

i'm pretty sure tags cannot have `/` in them, so we'd need to sed out illegal chars